### ISSUE TITLE
Remote issue, crackling noise, no option for closing the game and higher bandwidth?

### ISSUE THREAD

**Issue by Karabacak021** [OneLiberty/moonlight-chrome-tizen#14]
Greetings! 
I appriciate the work for the app :) 

I just started using it today, but I found few issues. 

1. If I playing and accidentally a wrong button is pressed on the remote (Like shitty dedicated streaming buttons) sometimes the Moonlight no longer recognize the TV remote. 
2. Sometimes there is a crackling noise when there is a bit lagg. 
3. With the remote, there is no option to close the gaming, only to leave the moonlight with the back button, then going back to moonlight then pressing the "quit current app"
4. May can you add higher maximum bandwidth to 4k? I can't set higher then 100mbit.
5. My controller is not rumbling with moonlight
6. In XInput the buttons works well, but in DirectInput mode the buttons are wrongly assigned and it feels the "Button 10" is pressed continously. (I tried only in Tekken 7)

My things: 55Q60T, logitech F710 controller

**Comment by OneLiberty** [comment]
Hi ! 
Short answer before going on vacation. 

1. This is because you can "theoretically" use the remote to control the host PC. This is not practical at all, but might be useful in some case. Volume should still work tho. 
2. On my end i don't have any issue with audio. Is it specific to to Tizen port ? 
3. Indeed, there aren't any way to quit the stream from "only" the remote, I believe the keyboard shortcut is broken too. I'll be looking into that. 
4. Not sure it's needed as i don't recall seeing a TV that supports more than 100mbps ([source](https://developer.samsung.com/smarttv/develop/specifications/media-specifications/2023-tv-video-specifications.html))
5. Unfortunately some controllers aren't compatible with the rumble (i made a very small list of compatible controllers over [there](https://github.com/KyroFrCode/moonlight-chrome-tizen/pull/16)) and without [debug](https://github.com/OneLiberty/moonlight-chrome-tizen/discussions/9) logs nor access to the same controller that you have, it will be difficult to troubleshoot. 
6. Yeah, some controller doesn't seems to have the correct layout, i'd like to had remapping but i think this is too much work for me, i'm not confident enough to do that, i'll try tho. (Made a discussion about this [here](https://github.com/OneLiberty/moonlight-chrome-tizen/discussions/13)). 

Thanks for the feedback, i'll keep you informed :)

**Comment by Karabacak021** [comment]
Thank you for the answer, have a good vacation!

1. The first issue main problem is, if the Moonlight no longer recognize it, there is no option to close moonlight connection, only with restart. 
2. Yes, in other apps like streamings and steam thing, there is no sounds like that, what I found about it, that it is not constant, only happen with sudden sound effects, like the sound of hooves in red dead redemption 2. On the source PC there isn't any crackling.
3. thanks :) 
4. Got it.
5. I just tested with more games, it is working, sorry!

**Comment by OneLiberty** [comment]
@Karabacak021 kinda forget about that. 
I pushed an update which allow the red button of your remote to close the stream.
did it resolve your issue with the remote ? 

Cracking sound might be due to TV max internet bandwidth being saturated. I don't recommend using a video bitrate higher than 60. (Ethernet is 100/100mpbs, wifi is slightly higher on some tvs).

**Comment by Karabacak021** [comment]
Thank you! I will test it! :) 

About the cracking sounds:
Unfortunately, the clicking sounds were present at 60, but even at 40, but I recently switched from GameStream to Sunshine, which solved 99% of the problem. Now the clicks are only present for the first few seconds of the broadcast and then disappear (but this is only the result of 2 days of testing)

I'm sure it is not the bandwidth wideness, because I'm on AX wifi (Only the tv is AC)
I measured the next with OpenSpeedTest Server running on my PC same as Sunshine:
few test runs on the TV standard browser: 170-197Mbps download, 240-260Mbps Upload, 2.5-5.0ms ping, jitter: 0.4-0.6ms
Interesting that the download is slower, maybe this is the max write speed of the tv storage.
